samples/RoutingSample.Web/UseEndpointRoutingStartup.cs

```diff
@@ -77,6 +80,21 @@ public void ConfigureServices(IServiceCollection services)
                         0,
                         EndpointMetadataCollection.Empty,
                         "withoptionalconstraints"),
+                    new MatcherEndpoint((next) => (httpContext) =>
+                    {
+                        using (var writer = new StreamWriter(httpContext.Response.Body, Encoding.UTF8, 1024, leaveOpen: true))
+                        {
+                            var graphWriter = httpContext.RequestServices.GetRequiredService<DfaGraphWriter>();
+                            var dataSource = httpContext.RequestServices.GetRequiredService<CompositeEndpointDataSource>();
+                            graphWriter.Write(dataSource, writer);
+                        }
+
+                        return Task.CompletedTask;
```

Comment:
I started trying to add a page that would render the graph and quickly got lost in the morass of NPM+dotnet. I don't want to spend too much time on that now.